Automattic/wp-calypso
# Show Welcome Tour to all new users on Desktop, NUX to users on Mobile

## Changes proposed in this Pull Request

- [x] Remove `apply_filter` code that hooked to experiment
- [x] Remove client code that handled the variant variable and determined which editor welcome to show to user
- [x] Remove  check fo Anchor.fm usage since  @allilevine has [REMOVED anchor NUX content](https://github.com/Automattic/wp-calypso/commit/1c2161f5e768d87353d42f9e4ad2ad3d36a06b36)
- [x] Show NUX modal to mobile users

A subsequent diff will remove the experiment code D53894-code

## Testing instructions

* pull down this branch
* `yarn dev --sync` to your sandbox
 OR
* from your sandbox: 
  * `cd /home/wpcom/public_html`  
  *  `install-plugin.sh etk update/show-welcome-tour-to-new-users-cept-anchorfm`  (nifty!)
* On a simple site that is sandboxed test the following:

* Clear application data:  In dev tools Application tab you must clear the site's local storage to ensure the @automattic/nux store is reset. Must be done in order to show the Welcome Guide on first load.

**Scenario #1 Desktop**
* Welcome Tour should show when opening editor
  * go through all slides, minimize, maximize, etc to make sure Tour works 
* Open Tour from Options Menu (aka More Menu) -> "Welcome Guide"
* Close, then re-open editor and make sure the Welcome Tour does not automatically show

**Scenario #2 Mobile**
* Clear application data agin
* Use Chrome dev tools to mimic mobile
 OR
* BETTER if you are using https://github.com/Automattic/a8c-wp-env you can easily test on your mobile device via the Jurassic Tube integration (or ngrok)
* The NUX Modal should show


Related to # https://github.com/Automattic/wp-calypso/issues/50877
